app/grandchallenge/core/static/js/jsoneditor_widget.mjs

```diff
@@ -11,26 +11,41 @@ function initialize_jsoneditor_widget(jsoneditorWidgetID) {
         jsoneditorWidget.classList.remove("d-none");
     } else {
         const options = {
-            mode: "tree",
+            mode: "code",
```

Comment:
Lets keep it defaulting to tree mode right now. You are a developer so might switch straight away, but the users are not and are more often just looking at the values of the field. The tree mode offers a nicer interface for that.

```suggestion
            mode: "tree",
```